.github/workflows/main.yml

```diff
@@ -88,8 +92,55 @@ jobs:
       - uses: actions/checkout@v1
       - uses: psf/black@stable
       - name: Build
-        run: pip3 install -e . -r requirements/test.txt
+        run: |
+          pip3 install --user --upgrade pip
+          pip3 install -e . -r requirements/test.txt
       - name: Run tests
         run: pytest
         env:
           DB: postgres
+
+  test-oracle:
+    name: test-python${{ matrix.python-version }}-oracle
+    strategy:
+      matrix:
+        python-version:
+         - "3.6"
+         - "3.10"
+    services:
+      oracle:
+        image: gvenzl/oracle-xe
```

Comment:
Which version of oracle does this use ?
Can we use the oldest version available ?

Reply:
It was currently 21, downgraded to 18 for now.